arviz/tests/base_tests/test_data.py

```diff
@@ -480,6 +480,28 @@ def test_repr_html(self):
         assert escape(idata.__repr__()) in html
         xr.set_options(display_style=display_style)
 
+    def test_add_groups(self):
```

Comment:
I would split that into several functions, one function to test "normal" behaviour, one for error, another for warning...